app/Platform/Models/Game.php

```diff
@@ -161,6 +161,24 @@ public function shouldBeSearchable(): bool
         return false;
     }
 
+    // == temp subset detection
+
+    public function getParentGameId(string $title, int $consoleID, int $gameID): ?int
+    {
+
+        $result = Game::where('Title', 'LIKE', '%' . $title . '%')->where('ConsoleID', '=', $consoleID)->first();
+
+        if ($result->Title = $title) {
```

Comment:
Is this a logic error?

Reply:
`===` 🙏

Reply:
I'm guessing this needs updating elsewhere as well?

Reply:
Seems to be the only occurrence in this PR - haven't checked the other files yet. I hope we didn't miss any 😅